### PR TITLE
work for namer plugin

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,11 @@ See documentation [here](doc/00-overview.md)
 
 ``` yaml
 use-extension:
-  "@autorest/cli.common": "latest"
+  "@autorest/cli.common": "/home/qiaozha/code/autorest.cli.common"
```
Comment (zikalino) [inline README.md]: seems it's your local path

File: src/plugins/aznamer.ts [hunk 2]
```diff
@@ -1,37 +1,56 @@
...
+    codeModel: CodeModel;
 
     constructor(protected session: Session<CodeModel>) {
         this.codeModel = session.model;
     }
 
-    async init() {
-        // any configuration if necessary
-        return this;
+    changeCamelToDash(str: string) {
```
Comment (zikalino) [inline src/plugins/aznamer.ts]: we should add this in some kind of common helpers file for entire extension.
doesn't such function exist in any common autorest libraries?

File: README.md (hunk 1 above)
Comment (qiaozha) [reply README.md]: yes, but cli.common has not released yet. can you release it first

File: src/plugins/aznamer.ts (hunk 2 above)
Comment (qiaozha) [reply src/plugins/aznamer.ts]: I agree.

**Comment by qiaozha** [comment]
@zikalino  changed the changeCamelToDash to a common helper please review

**Pull request merged**